activesupport/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 *   Add `String#upcase_first` method.
 
-     *Glauco Custódio*
+    *Glauco Custódio*, *bogdanvlviv*
```

Comment:
This change is also not needed. We don't add changelog entries for bug fixes in the features.